app/templates/_package.json

```diff
@@ -62,6 +62,8 @@
     "karma-phantomjs-launcher": "0.2.0",
     "phantomjs": "1.9.17",
     "karma": "0.12.35",
+    "karma-coverage": "^0.4.2",
```

Comment:
Please use fixed versions, like "0.4.2", we've had too many issues with this (I know, I need to do a "policies" document for all this)